adform-htb.js

```diff
@@ -144,9 +218,36 @@ function AdformHtb(configs) {
         var queryObj = {};
         var callbackId = System.generateUniqueId();
 
-        /* Change this to your bidder endpoint.*/
-        var baseUrl = Browser.getProtocol() + '//someAdapterEndpoint.com/bid';
+        var request = [], item, _key, _value, i, j, k;
 
+        var globalParams = [ [ 'adxDomain', 'adx.adform.net' ], [ 'fd', 1 ], [ 'url', null ], [ 'tid', null ] ];
```

Comment:
Is there a need to declare this globalParam? From the code below, a lot of these can be coded in the request url directly.